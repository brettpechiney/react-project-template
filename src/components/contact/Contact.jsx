import React from 'react';
import { hot } from 'react-hot-loader/root';
import './contact.scss';

const Contact = () => (
  <section className="contact--container">
    <p className="contact-intro">This is the contact page.</p>
  </section>
);

export default hot(Contact);
