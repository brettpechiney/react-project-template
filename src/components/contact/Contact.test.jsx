import React from 'react';
import { shallow } from 'enzyme';
import Contact from './Contact';

const setup = propOverrides => {
  const props = Object.assign({}, propOverrides);

  const wrapper = shallow(<Contact {...props} />);

  return {
    props,
    wrapper,
  };
};

describe('<Contact />', () => {
  const { wrapper } = setup();

  it('renders', () => {
    expect(wrapper.exists()).toBe(true);
  });
});
