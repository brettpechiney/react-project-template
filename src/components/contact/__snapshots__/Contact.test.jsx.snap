// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact /> renders correctly 1`] = `
<section
  className="src-components-contact-___contact__contact___2-wh2"
>
  <p
    className="src-components-contact-___contact__intro___3TQFO"
  >
    This is the contact page.
  </p>
</section>
`;
